### DISCUSSION TITLE
sungwoncho/meteor-mailman

### DISCUSSION THREAD

**Discussion by juliomac** [meteor/meteor#11935] [Compat Packages]
I love this little package [https://github.com/sungwoncho/meteor-mailman](https://github.com/sungwoncho/meteor-mailman).

It is a very simple one, just a few lines of code. But solves a big problem for me while developing.

It should not be difficult to uptake it. Probably just loosing its dependency on a fixed version of  email@1.0.8 would already be enough.

**Comment by StorytellerCZ** [comment]
Do you want someone to submit a PR to update it or for it to become a core package or to be taken up by Meteor Community packages?

**Reply by juliomac** [reply]
I would be nice if it becomes a Meteor Community package or core package.

**Reply by StorytellerCZ** [reply]
For MCP, please use this form to get things started: https://github.com/Meteor-Community-Packages/organization/issues/new?assignees=&labels=&template=new_package.md&title=New+package+for+inclusion+-
